quickstart_docker/README.md

```diff
@@ -0,0 +1,51 @@
+# Delta Lake Quickstart Container
+
+A docker deployment of Spark with Delta Lake
+
+
+## Docker High Level Instructions
+
+- clone this repo onto a resource running docker engine
+- navigate to the cloned folder
+- use the commands in this file to build the image, run the container, and navigate to the Jupyter Lab service
+- open and follow the quickstart notebook and/or exec into a container shell and follow scala/python instructions there and/or open a terminal in jupyter hub and follow the scala/python instructions there
+
+## Docker build/run Instructions
+
+- open a bash shell (if on windows use git bash, WSL, or any shell configged for bash commands)
+- execute build command from same folder as docker file
+- exectute run command
+- get the log tail and navigate to the last url displayed
+
+## Docker Commands
+
+### Build
+
+```bash
+cd quickstart_docker
+docker build -t delta_quickstart -f Dockerfile_quickstart .
+```
+
+### Run
+
+```bash
+docker run -d --rm -p 8888-8889:8888-8889 delta_quickstart 
+```
+
+### Exec into a shell
+```bash
+docker exec -it <first 4 containerid> bash
+```
+
```

Comment:
This is awesome, could you also include a quick example of how to get it up and running?  For example:

Start your PySpark Shell with Delta 2.1. and Spark 3.3
`pyspark --packages io.delta:delta-core_2.12:2.1.0  --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension"  --conf "spark.sql.catalog.spark_catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog"`

or use the `startup.sh` command that is included in this docker container.

Within your `pyspark` shell, run the following commands to create and read a simple Delta table generated by a Spark dataframe.
```
from delta import *

— Create Spark DataFrame
data = spark.range(0, 5)
data.write.format("delta").save("/tmp/delta-table")

— Read Delta table
df = spark.read.format("delta").load("/tmp/delta-table")                    

— Show Delta table
df.show()
```

To verify that you have a Delta table, you can list the contents within the folder of your Delta table.  For example, in the previous code, you saved the table in `/tmp/delta-table`.  Once you close your pyspark process, run a list command in your Docker shell and you should get something similar to below.

```
$ ls -lsgA /tmp/delta-table
total 36
4 drwxr-xr-x 2 NBuser 4096 Oct 18 02:02 _delta_log
4 -rw-r--r-- 1 NBuser  478 Oct 18 02:02 part-00000-b968d89a-b299-401f-a6db-ba0c160633ab-c000.snappy.parquet
4 -rw-r--r-- 1 NBuser   12 Oct 18 02:02 .part-00000-b968d89a-b299-401f-a6db-ba0c160633ab-c000.snappy.parquet.crc
4 -rw-r--r-- 1 NBuser  478 Oct 18 02:02 part-00001-f0f8ea27-b522-4c2c-8fe3-7224fccacb91-c000.snappy.parquet
4 -rw-r--r-- 1 NBuser   12 Oct 18 02:02 .part-00001-f0f8ea27-b522-4c2c-8fe3-7224fccacb91-c000.snappy.parquet.crc
4 -rw-r--r-- 1 NBuser  478 Oct 18 02:02 part-00002-b8a1ea0d-0637-4432-8ab6-8ec864edb6b0-c000.snappy.parquet
4 -rw-r--r-- 1 NBuser   12 Oct 18 02:02 .part-00002-b8a1ea0d-0637-4432-8ab6-8ec864edb6b0-c000.snappy.parquet.crc
4 -rw-r--r-- 1 NBuser  486 Oct 18 02:02 part-00003-ba20f466-8cb6-4827-9c10-218e8933f0f7-c000.snappy.parquet
4 -rw-r--r-- 1 NBuser   12 Oct 18 02:02 .part-00003-ba20f466-8cb6-4827-9c10-218e8933f0f7-c000.snappy.parquet.crc
```

For more information, please refer to [Delta Lake Getting Started](https://delta.io/learn/getting-started)

Reply:
great suggestion, yes I can do that.

Reply:
this is done

Reply:
this is done

Reply:
wunderbar!